js/contentForeground/contentWeVoteUI.js

```diff
@@ -1469,6 +1482,14 @@ function orgChoiceDialog (orgList) {
   });
 }
 
+async function openAdminPage () {
+  const { chrome: { runtime: { sendMessage, lastError } } } = window;
+  debugFgLog('ENTERING openAdminPage');
+  const state = await getGlobalState();
+  const { organizationWeVoteId, voterGuidePossibilityId, voterWeVoteId } = state;
+  window.open('https://api.wevoteusa.org/vg/create/?voter_guide_possibility_id=8852');
```

Comment:
Hi @stilwellmg you still have a hardcoded variable I think you want to replace with `voterGuidePossibilityId`

Reply:
Fixed!